README.md

```diff
@@ -9,6 +9,43 @@
 
 > The properties will be taken from the 'gradle.properties' file.
 
+- Command line parameters
```

Comment:
To keep this file clean, move it to a wiki page with the title "EtendoRX Advanced options" and replace it with the corresponding link.